dist/index.html

```diff
@@ -17,15 +18,15 @@
     <title>Leaderboard List App</title>
   <script defer src="runtime.bundle.js"></script><script defer src="index.bundle.js"></script></head>
 
-  <body>
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍

Reply:
I agree. I might make a logo in the future. Thanks for your kind advice. :)